wallet/httpclient.go

```diff
@@ -140,6 +144,23 @@ func newHTTPClient(cfg *htpc) (*http.Client, error) {
 	return &client, nil
 }
 
+func (cfg *htpc) CheckSyncUpdate(localheight int64) (bool, error) {
```

Comment:
`htpc` is not a good  `golang` type name.  please see https://github.com/golang/go/wiki/CodeReviewComments